<?xml version="1.0" encoding="utf-8"?>
<test-run id="2" name="Incapsulation.RationalNumbers.exe" fullname="C:/Users/vadim/OneDrive/Рабочий стол/подготовка к пересдачам/юлёрн ипаный/инкапсуляция/Рациональные числа/bin/Debug/Incapsulation.RationalNumbers.exe" testcasecount="48" result="Failed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.246507" total="48" passed="32" failed="16" inconclusive="0" skipped="0" asserts="69" random-seed="543921520">
  <command-line><![CDATA["C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\bin\Debug\Incapsulation.RationalNumbers.exe"   ]]></command-line>
  <filter />
  <test-suite type="Assembly" id="1053" name="Incapsulation.RationalNumbers.exe" fullname="C:/Users/vadim/OneDrive/Рабочий стол/подготовка к пересдачам/юлёрн ипаный/инкапсуляция/Рациональные числа/bin/Debug/Incapsulation.RationalNumbers.exe" runstate="Runnable" testcasecount="48" result="Failed" site="Child" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.246507" total="48" passed="32" failed="16" warnings="0" inconclusive="0" skipped="0" asserts="69">
    <environment framework-version="3.6.0.0" clr-version="4.0.30319.42000" os-version="Microsoft Windows NT 6.2.9200.0" platform="Win32NT" cwd="C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\bin\Debug" machine-name="DESKTOP-RBCI3Q1" user="vadim" user-domain="DESKTOP-RBCI3Q1" culture="ru-RU" uiculture="ru-RU" os-architecture="x86" />
    <settings>
      <setting name="WorkDirectory" value="C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\bin\Debug" />
      <setting name="NumberOfTestWorkers" value="12" />
    </settings>
    <properties>
      <property name="_PID" value="2400" />
      <property name="_APPDOMAIN" value="Incapsulation.RationalNumbers.exe" />
    </properties>
    <failure>
      <message><![CDATA[One or more child tests had errors]]></message>
    </failure>
    <test-suite type="TestSuite" id="1054" name="Incapsulation" fullname="Incapsulation" runstate="Runnable" testcasecount="48" result="Failed" site="Child" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.238074" total="48" passed="32" failed="16" warnings="0" inconclusive="0" skipped="0" asserts="69">
      <failure>
        <message><![CDATA[One or more child tests had errors]]></message>
      </failure>
      <test-suite type="TestSuite" id="1055" name="RationalNumbers" fullname="Incapsulation.RationalNumbers" runstate="Runnable" testcasecount="48" result="Failed" site="Child" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.235061" total="48" passed="32" failed="16" warnings="0" inconclusive="0" skipped="0" asserts="69">
        <failure>
          <message><![CDATA[One or more child tests had errors]]></message>
        </failure>
        <test-suite type="TestFixture" id="1000" name="Rational_should" fullname="Incapsulation.RationalNumbers.Rational_should" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" testcasecount="48" result="Failed" site="Child" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.231536" total="48" passed="32" failed="16" warnings="0" inconclusive="0" skipped="0" asserts="69">
          <failure>
            <message><![CDATA[One or more child tests had errors]]></message>
          </failure>
          <test-case id="1004" name="BeCorrectWithZeroNumerator" fullname="Incapsulation.RationalNumbers.Rational_should.BeCorrectWithZeroNumerator" methodname="BeCorrectWithZeroNumerator" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="2036061555" result="Failed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.048543" asserts="3">
            <failure>
              <message><![CDATA[  Expected: 1
  But was:  5
]]></message>
              <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 13
в Incapsulation.RationalNumbers.Rational_should.BeCorrectWithZeroNumerator() в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 37
]]></stack-trace>
            </failure>
            <assertions>
              <assertion result="Failed">
                <message><![CDATA[  Expected: 1
  But was:  5
]]></message>
                <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 13
в Incapsulation.RationalNumbers.Rational_should.BeCorrectWithZeroNumerator() в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 37
]]></stack-trace>
              </assertion>
            </assertions>
          </test-case>
          <test-case id="1032" name="ConvertFromInt" fullname="Incapsulation.RationalNumbers.Rational_should.ConvertFromInt" methodname="ConvertFromInt" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="32448870" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000229" asserts="3" />
          <test-suite type="ParameterizedMethod" id="1031" name="ConvertToDouble" fullname="Incapsulation.RationalNumbers.Rational_should.ConvertToDouble" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" testcasecount="6" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.020811" total="6" passed="6" failed="0" warnings="0" inconclusive="0" skipped="0" asserts="6">
            <test-case id="1025" name="ConvertToDouble(1,2,0.5d)" fullname="Incapsulation.RationalNumbers.Rational_should.ConvertToDouble(1,2,0.5d)" methodname="ConvertToDouble" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1003223385" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.001748" asserts="1" />
            <test-case id="1026" name="ConvertToDouble(10,5,2.0d)" fullname="Incapsulation.RationalNumbers.Rational_should.ConvertToDouble(10,5,2.0d)" methodname="ConvertToDouble" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="312914902" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000061" asserts="1" />
            <test-case id="1027" name="ConvertToDouble(-1,5,-0.2d)" fullname="Incapsulation.RationalNumbers.Rational_should.ConvertToDouble(-1,5,-0.2d)" methodname="ConvertToDouble" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1873567489" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000132" asserts="1" />
            <test-case id="1028" name="ConvertToDouble(10,0,double.NaN)" fullname="Incapsulation.RationalNumbers.Rational_should.ConvertToDouble(10,0,double.NaN)" methodname="ConvertToDouble" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1686190864" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000071" asserts="1" />
            <test-case id="1029" name="ConvertToDouble(-10,0,double.NaN)" fullname="Incapsulation.RationalNumbers.Rational_should.ConvertToDouble(-10,0,double.NaN)" methodname="ConvertToDouble" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1549472295" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000082" asserts="1" />
            <test-case id="1030" name="ConvertToDouble(0,0,double.NaN)" fullname="Incapsulation.RationalNumbers.Rational_should.ConvertToDouble(0,0,double.NaN)" methodname="ConvertToDouble" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1582969357" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000128" asserts="1" />
          </test-suite>
          <test-case id="1022" name="Divide" fullname="Incapsulation.RationalNumbers.Rational_should.Divide" methodname="Divide" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1258005453" result="Failed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.001957" asserts="2">
            <failure>
              <message><![CDATA[  Expected: -1
  But was:  2
]]></message>
              <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.Divide() в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 97
]]></stack-trace>
            </failure>
            <assertions>
              <assertion result="Failed">
                <message><![CDATA[  Expected: -1
  But was:  2
]]></message>
                <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.Divide() в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 97
]]></stack-trace>
              </assertion>
            </assertions>
          </test-case>
          <test-case id="1024" name="DivideToZero" fullname="Incapsulation.RationalNumbers.Rational_should.DivideToZero" methodname="DivideToZero" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1144651387" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000619" asserts="1" />
          <test-case id="1023" name="DivideWithNan" fullname="Incapsulation.RationalNumbers.Rational_should.DivideWithNan" methodname="DivideWithNan" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1563568112" result="Failed" label="Error" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.001909" asserts="0">
            <failure>
              <message><![CDATA[System.Exception : Выдано исключение типа "System.Exception".]]></message>
              <stack-trace><![CDATA[   в Incapsulation.RationalNumbers.Rational.op_Division(Rational r1, Rational r2) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational.cs:строка 58
   в Incapsulation.RationalNumbers.Rational_should.DivideWithNan() в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 103]]></stack-trace>
            </failure>
          </test-case>
          <test-suite type="ParameterizedMethod" id="1045" name="ExplicitlyConvertToInt" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToInt" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" testcasecount="12" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.036528" total="12" passed="12" failed="0" warnings="0" inconclusive="0" skipped="0" asserts="12">
            <test-case id="1033" name="ExplicitlyConvertToInt(0,1,0)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToInt(0,1,0)" methodname="ExplicitlyConvertToInt" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1983013793" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000282" asserts="1" />
            <test-case id="1034" name="ExplicitlyConvertToInt(1,1,1)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToInt(1,1,1)" methodname="ExplicitlyConvertToInt" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="99327298" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000071" asserts="1" />
            <test-case id="1035" name="ExplicitlyConvertToInt(2,1,2)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToInt(2,1,2)" methodname="ExplicitlyConvertToInt" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1945743878" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000144" asserts="1" />
            <test-case id="1036" name="ExplicitlyConvertToInt(3,1,3)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToInt(3,1,3)" methodname="ExplicitlyConvertToInt" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="2013640540" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000083" asserts="1" />
            <test-case id="1037" name="ExplicitlyConvertToInt(2,2,1)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToInt(2,2,1)" methodname="ExplicitlyConvertToInt" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="984683368" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000084" asserts="1" />
            <test-case id="1038" name="ExplicitlyConvertToInt(6,3,2)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToInt(6,3,2)" methodname="ExplicitlyConvertToInt" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1153092876" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000162" asserts="1" />
            <test-case id="1039" name="ExplicitlyConvertToInt(12,2,6)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToInt(12,2,6)" methodname="ExplicitlyConvertToInt" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1181251814" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000096" asserts="1" />
            <test-case id="1040" name="ExplicitlyConvertToInt(12,3,4)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToInt(12,3,4)" methodname="ExplicitlyConvertToInt" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="518169575" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000079" asserts="1" />
            <test-case id="1041" name="ExplicitlyConvertToInt(12,4,3)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToInt(12,4,3)" methodname="ExplicitlyConvertToInt" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="185604460" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000086" asserts="1" />
            <test-case id="1042" name="ExplicitlyConvertToInt(12,6,2)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToInt(12,6,2)" methodname="ExplicitlyConvertToInt" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="862507916" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000093" asserts="1" />
            <test-case id="1043" name="ExplicitlyConvertToInt(12,12,1)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToInt(12,12,1)" methodname="ExplicitlyConvertToInt" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1342697696" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000103" asserts="1" />
            <test-case id="1044" name="ExplicitlyConvertToInt(1000,1,1000)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToInt(1000,1,1000)" methodname="ExplicitlyConvertToInt" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1173781116" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000079" asserts="1" />
          </test-suite>
          <test-suite type="ParameterizedMethod" id="1052" name="ExplicitlyConvertToIntAndFailsIfNonConvertible" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToIntAndFailsIfNonConvertible" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" testcasecount="6" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.020003" total="6" passed="6" failed="0" warnings="0" inconclusive="0" skipped="0" asserts="6">
            <test-case id="1046" name="ExplicitlyConvertToIntAndFailsIfNonConvertible(1,2)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToIntAndFailsIfNonConvertible(1,2)" methodname="ExplicitlyConvertToIntAndFailsIfNonConvertible" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="291707983" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.001990" asserts="1" />
            <test-case id="1047" name="ExplicitlyConvertToIntAndFailsIfNonConvertible(12,5)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToIntAndFailsIfNonConvertible(12,5)" methodname="ExplicitlyConvertToIntAndFailsIfNonConvertible" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="2083996281" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000129" asserts="1" />
            <test-case id="1048" name="ExplicitlyConvertToIntAndFailsIfNonConvertible(12,10)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToIntAndFailsIfNonConvertible(12,10)" methodname="ExplicitlyConvertToIntAndFailsIfNonConvertible" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="516525163" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000112" asserts="1" />
            <test-case id="1049" name="ExplicitlyConvertToIntAndFailsIfNonConvertible(25,8)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToIntAndFailsIfNonConvertible(25,8)" methodname="ExplicitlyConvertToIntAndFailsIfNonConvertible" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1825087203" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000242" asserts="1" />
            <test-case id="1050" name="ExplicitlyConvertToIntAndFailsIfNonConvertible(2,3)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToIntAndFailsIfNonConvertible(2,3)" methodname="ExplicitlyConvertToIntAndFailsIfNonConvertible" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="260833924" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000172" asserts="1" />
            <test-case id="1051" name="ExplicitlyConvertToIntAndFailsIfNonConvertible(2,4)" fullname="Incapsulation.RationalNumbers.Rational_should.ExplicitlyConvertToIntAndFailsIfNonConvertible(2,4)" methodname="ExplicitlyConvertToIntAndFailsIfNonConvertible" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1644695024" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000140" asserts="1" />
          </test-suite>
          <test-suite type="ParameterizedMethod" id="1015" name="InitializeAndReduce1" fullname="Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" testcasecount="10" result="Failed" site="Child" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.039012" total="10" passed="2" failed="8" warnings="0" inconclusive="0" skipped="0" asserts="22">
            <failure>
              <message><![CDATA[One or more child tests had errors]]></message>
            </failure>
            <test-case id="1005" name="InitializeAndReduce1(1,2,2,4)" fullname="Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(1,2,2,4)" methodname="InitializeAndReduce1" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1095901921" result="Failed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000988" asserts="2">
              <failure>
                <message><![CDATA[  Expected: 1
  But was:  2
]]></message>
                <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
              </failure>
              <assertions>
                <assertion result="Failed">
                  <message><![CDATA[  Expected: 1
  But was:  2
]]></message>
                  <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
                </assertion>
              </assertions>
            </test-case>
            <test-case id="1006" name="InitializeAndReduce1(-1,2,-2,4)" fullname="Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(-1,2,-2,4)" methodname="InitializeAndReduce1" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1988828899" result="Failed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.001512" asserts="2">
              <failure>
                <message><![CDATA[  Expected: -1
  But was:  -2
]]></message>
                <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
              </failure>
              <assertions>
                <assertion result="Failed">
                  <message><![CDATA[  Expected: -1
  But was:  -2
]]></message>
                  <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
                </assertion>
              </assertions>
            </test-case>
            <test-case id="1007" name="InitializeAndReduce1(-1,2,2,-4)" fullname="Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(-1,2,2,-4)" methodname="InitializeAndReduce1" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="9609748" result="Failed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.001136" asserts="2">
              <failure>
                <message><![CDATA[  Expected: -1
  But was:  2
]]></message>
                <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
              </failure>
              <assertions>
                <assertion result="Failed">
                  <message><![CDATA[  Expected: -1
  But was:  2
]]></message>
                  <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
                </assertion>
              </assertions>
            </test-case>
            <test-case id="1008" name="InitializeAndReduce1(1,2,-2,-4)" fullname="Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(1,2,-2,-4)" methodname="InitializeAndReduce1" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1030861611" result="Failed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.001592" asserts="2">
              <failure>
                <message><![CDATA[  Expected: 1
  But was:  -2
]]></message>
                <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
              </failure>
              <assertions>
                <assertion result="Failed">
                  <message><![CDATA[  Expected: 1
  But was:  -2
]]></message>
                  <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
                </assertion>
              </assertions>
            </test-case>
            <test-case id="1009" name="InitializeAndReduce1(1,2,1,2)" fullname="Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(1,2,1,2)" methodname="InitializeAndReduce1" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="160560735" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000133" asserts="3" />
            <test-case id="1010" name="InitializeAndReduce1(1,2,8,16)" fullname="Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(1,2,8,16)" methodname="InitializeAndReduce1" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1469424698" result="Failed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000871" asserts="2">
              <failure>
                <message><![CDATA[  Expected: 1
  But was:  8
]]></message>
                <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
              </failure>
              <assertions>
                <assertion result="Failed">
                  <message><![CDATA[  Expected: 1
  But was:  8
]]></message>
                  <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
                </assertion>
              </assertions>
            </test-case>
            <test-case id="1011" name="InitializeAndReduce1(2,3,10,15)" fullname="Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(2,3,10,15)" methodname="InitializeAndReduce1" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1274231303" result="Failed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000756" asserts="2">
              <failure>
                <message><![CDATA[  Expected: 2
  But was:  10
]]></message>
                <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
              </failure>
              <assertions>
                <assertion result="Failed">
                  <message><![CDATA[  Expected: 2
  But was:  10
]]></message>
                  <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
                </assertion>
              </assertions>
            </test-case>
            <test-case id="1012" name="InitializeAndReduce1(4,7,16,28)" fullname="Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(4,7,16,28)" methodname="InitializeAndReduce1" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="736825174" result="Failed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000914" asserts="2">
              <failure>
                <message><![CDATA[  Expected: 4
  But was:  16
]]></message>
                <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
              </failure>
              <assertions>
                <assertion result="Failed">
                  <message><![CDATA[  Expected: 4
  But was:  16
]]></message>
                  <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
                </assertion>
              </assertions>
            </test-case>
            <test-case id="1013" name="InitializeAndReduce1(3,256,12,1024)" fullname="Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(3,256,12,1024)" methodname="InitializeAndReduce1" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1281720558" result="Failed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.001017" asserts="2">
              <failure>
                <message><![CDATA[  Expected: 3
  But was:  12
]]></message>
                <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
              </failure>
              <assertions>
                <assertion result="Failed">
                  <message><![CDATA[  Expected: 3
  But was:  12
]]></message>
                  <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(Int32 expectedNum, Int32 expectedDen, Int32 num, Int32 den) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 52
]]></stack-trace>
                </assertion>
              </assertions>
            </test-case>
            <test-case id="1014" name="InitializeAndReduce1(1,1,1,1)" fullname="Incapsulation.RationalNumbers.Rational_should.InitializeAndReduce1(1,1,1,1)" methodname="InitializeAndReduce1" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1183280450" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000065" asserts="3" />
          </test-suite>
          <test-case id="1001" name="InitializeSimpleRatioCorrectly" fullname="Incapsulation.RationalNumbers.Rational_should.InitializeSimpleRatioCorrectly" methodname="InitializeSimpleRatioCorrectly" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1473397264" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000117" asserts="3" />
          <test-case id="1002" name="InitializeWithoutDenominator" fullname="Incapsulation.RationalNumbers.Rational_should.InitializeWithoutDenominator" methodname="InitializeWithoutDenominator" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="768336291" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000195" asserts="3" />
          <test-case id="1003" name="InitializeWithZeroDenominator" fullname="Incapsulation.RationalNumbers.Rational_should.InitializeWithZeroDenominator" methodname="InitializeWithZeroDenominator" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1792848393" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000301" asserts="1" />
          <test-case id="1020" name="Multiply" fullname="Incapsulation.RationalNumbers.Rational_should.Multiply" methodname="Multiply" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="820186309" result="Passed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000187" asserts="3" />
          <test-case id="1021" name="MultiplyWithNan" fullname="Incapsulation.RationalNumbers.Rational_should.MultiplyWithNan" methodname="MultiplyWithNan" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1734481067" result="Failed" label="Error" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000701" asserts="0">
            <failure>
              <message><![CDATA[System.Exception : Выдано исключение типа "System.Exception".]]></message>
              <stack-trace><![CDATA[   в Incapsulation.RationalNumbers.Rational.op_Multiply(Rational r1, Rational r2) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational.cs:строка 50
   в Incapsulation.RationalNumbers.Rational_should.MultiplyWithNan() в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 90]]></stack-trace>
            </failure>
          </test-case>
          <test-case id="1018" name="Subtract" fullname="Incapsulation.RationalNumbers.Rational_should.Subtract" methodname="Subtract" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1519747782" result="Failed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.001451" asserts="2">
            <failure>
              <message><![CDATA[  Expected: 1
  But was:  2
]]></message>
              <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.Subtract() в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 71
]]></stack-trace>
            </failure>
            <assertions>
              <assertion result="Failed">
                <message><![CDATA[  Expected: 1
  But was:  2
]]></message>
                <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.Subtract() в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 71
]]></stack-trace>
              </assertion>
            </assertions>
          </test-case>
          <test-case id="1019" name="SubtractWithNan" fullname="Incapsulation.RationalNumbers.Rational_should.SubtractWithNan" methodname="SubtractWithNan" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1588645222" result="Failed" label="Error" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000879" asserts="0">
            <failure>
              <message><![CDATA[System.Exception : Выдано исключение типа "System.Exception".]]></message>
              <stack-trace><![CDATA[   в Incapsulation.RationalNumbers.Rational.op_Subtraction(Rational r1, Rational r2) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational.cs:строка 42
   в Incapsulation.RationalNumbers.Rational_should.SubtractWithNan() в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 77]]></stack-trace>
            </failure>
          </test-case>
          <test-case id="1016" name="Sum" fullname="Incapsulation.RationalNumbers.Rational_should.Sum" methodname="Sum" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="1767681940" result="Failed" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.001650" asserts="2">
            <failure>
              <message><![CDATA[  Expected: 1
  But was:  8
]]></message>
              <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.Sum() в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 58
]]></stack-trace>
            </failure>
            <assertions>
              <assertion result="Failed">
                <message><![CDATA[  Expected: 1
  But was:  8
]]></message>
                <stack-trace><![CDATA[в Incapsulation.RationalNumbers.Rational_should.AssertEqual(Int32 expectedNumerator, Int32 expectedDenominator, Rational actual) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 12
в Incapsulation.RationalNumbers.Rational_should.Sum() в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 58
]]></stack-trace>
              </assertion>
            </assertions>
          </test-case>
          <test-case id="1017" name="SumWithNan" fullname="Incapsulation.RationalNumbers.Rational_should.SumWithNan" methodname="SumWithNan" classname="Incapsulation.RationalNumbers.Rational_should" runstate="Runnable" seed="273673140" result="Failed" label="Error" start-time="2023-10-03 15:23:54Z" end-time="2023-10-03 15:23:54Z" duration="0.000660" asserts="0">
            <failure>
              <message><![CDATA[System.Exception : Выдано исключение типа "System.Exception".]]></message>
              <stack-trace><![CDATA[   в Incapsulation.RationalNumbers.Rational.op_Addition(Rational r1, Rational r2) в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational.cs:строка 34
   в Incapsulation.RationalNumbers.Rational_should.SumWithNan() в C:\Users\vadim\OneDrive\Рабочий стол\подготовка к пересдачам\юлёрн ипаный\инкапсуляция\Рациональные числа\Rational_should.cs:строка 64]]></stack-trace>
            </failure>
          </test-case>
        </test-suite>
      </test-suite>
    </test-suite>
  </test-suite>
</test-run>